tests/server/endpoints/test_orders_endpoint.py

```diff
@@ -1,62 +1,18 @@
 from http import HTTPStatus
 
 import mock.mock
-import pytest
 from flask.testing import FlaskClient
 
 from cg.apps.tb import TrailblazerAPI
 from cg.apps.tb.dto.summary_response import AnalysisSummary
-from cg.constants import Workflow
-from cg.services.orders.order_service.utils import create_order_response
-from cg.services.orders.order_status_service.dto.order_status_summary import (
-    OrderSummary,
-)
-from cg.services.orders.order_status_service.utils import create_summaries
 from cg.store.models import Order
 
 
-@pytest.mark.parametrize(
-    "limit, workflow, expected_orders",
-    [
-        (None, Workflow.MIP_DNA, 2),
-        (1, Workflow.MIP_DNA, 1),
-        (2, Workflow.MIP_DNA, 2),
-        (None, Workflow.BALSAMIC, 1),
-        (None, Workflow.FLUFFY, 0),
-        (None, None, 3),
-    ],
-)
-def test_orders_endpoint(
```

Comment:
I think we should drop this test because the exact same logic is covered below in another test. The alternative is to create a messy trailblazer response mock with the summaries.

Reply:
Do we test that the request and URI is parsed correctly in the other tests?

Reply:
I decided to go for integration tests on the service level instead, the API layer just calls the service so I don't think it adds value to keep

Reply:
Yeah, I guess that is fine.

Reply:
Re-added here: https://github.com/Clinical-Genomics/cg/pull/3050/commits/2c75b690026d7847aca899e152b8acb6fbe33fba